en/tutorials/land_detector.md

```diff
@@ -4,45 +4,49 @@ The land detector is a dynamic vehicle model representing key vehicle states suc
 
 ## Multicopter Land Detector Configuration
 
-The complete set of parameters is available in the QGroundControl parameter editor under the `LNDMC` prefix. The key parameters that might differ per airframe are these:
+The complete set of parameters is available in the *QGroundControl* parameter editor under the `LNDMC` prefix. The key parameters that might differ per airframe are these:
 
-* `LNDMC_MAN_DWNTHR` - the threshold \(in percent, default is 15%\) for how much manual throttle is allowed to be considering the state as landed. Systems with very high thrust-to-weight ratios like racers might need a lower setting here \(e.g. 8%\).
-* `MPC_THR_HOVER` - the hover throttle of the system \(in percent, default is 50%\). It is important to set this correctly as it does not only make the altitude control more accurate, but also ensures correct land detection. A racer or a big camera drone without payload mounted might need a much lower setting \(e.g. 35%\).
-* `MPC_THR_MIN` - the overall minimum throttle of the system. This should be set to enable a controlled descend.
-* `LNDMC_THR_RANGE` - this is a scaling factor to define the range between min and hover throttle that gets accepted as landed. Example: If the minimum throttle is 0.1, the hover throttle is 0.5 and the range is 0.2 \(20%\), then the highest throttle value that counts as landed is the min throttle plus \(0.5 - 0.1\) \* 0.2 = 0.08, so a total throttle of 0.18.
-* `LNDMC_POS_UPTHR` - throttle level to trigger takeoff \(in percent, default is 65%\). If the pilot raises above this threshold the system will attempt to take off. This value should be greater than the hover throttle.
+
+* [MPC_THR_HOVER](../advanced/parameter_reference.md#MPC_THR_HOVER) - the hover throttle of the system \(in percent, default is 50%\). It is important to set this correctly as it does not only make the altitude control more accurate, but also ensures correct land detection. A racer or a big camera drone without payload mounted might need a much lower setting \(e.g. 35%\).
+* [MPC_THR_MIN](../advanced/parameter_reference.md#MPC_THR_MIN) - the overall minimum throttle of the system. This should be set to enable a controlled descend.
+* [LNDMC_THR_RANGE](../advanced/parameter_reference.md#LNDMC_THR_RANGE) - this is a scaling factor to define the range between min and hover throttle that gets accepted as landed. Example: If the minimum throttle is 0.1, the hover throttle is 0.5 and the range is 0.2 \(20%\), then the highest throttle value that counts as landed is the min throttle plus \(0.5 - 0.1\) \* 0.2 = 0.08, so a total throttle of 0.18.
+
+The throttle level to trigger takeoff is hard-coded to 62.5%. If the pilot raises above this threshold the system will attempt to take off. This value should be greater than the hover throttle.
```

Comment:
This value of 62.5% is not actually connected to the throttle that the vehicle is generating, so it has nothing to do with hover throttle, I think.

It just means that if the stick is raised above 62.5% percent in altitude or position control, the controller will "do a takeoff". 

The logic is implemented in the position controller and not in the land detector, so that was the main change. It is now not the land detector detecting that the motors are spinning up and therefore saying "we're in air" which would tell the position controller to do a takeoff but instead the land detector will detect it based on the throttle output while the position controller looks at the manual input (stick).

Reply:
@julianoes OK, so I'll remove this comment from here altogether because it doesn't make sense in the context of land detection. Reasonable?
OK to merge?

This still looks useful info though. Would I be correct if in [Altitude mode](https://docs.px4.io/en/flight_modes/altitude.html) I added a line: 

> The vehicle will take off if the stick is raised above 62.5% percent.

When you say 62.5% you mean "off the full range from the throttle stick bottom position" right? So in effect, to just above the large dead zone in the middle?

Reply:
I think so but I'm not 100% sure.

Reply:
Thanks, I'll track that in https://github.com/PX4/px4_user_guide/issues/96